app/src/main/java/dao/IPokemonDAO.kt

```diff
@@ -5,7 +5,6 @@ import io.reactivex.Observable
 import retrofit2.http.GET
 
 interface IPokemonDAO {
-
```

Comment:
Removed blank first line of interface. This is done multiple times throughout the code review in classes also. This is not a big deal and is really just preference/best practice.